build.sbt

```diff
@@ -45,13 +47,16 @@ lazy val server = (project in file("server"))
       "org.webjars" % "jquery" % "2.2.4",
       "org.webjars" % "bootstrap" % "3.3.6" exclude("org.webjars", "jquery"),
       "org.monifu" %% "monifu" % monifuVer,
-
+      "org.json4s" %% "json4s-native" % "3.4.0",
```

Comment:
Play uses "org.asynchttpclient" % "async-http-client" % "2.0.20" underneath, we could use play's http client or asynchttpclient directly